src/main/scala/io/iohk/ethereum/transactions/PendingTransactionsManager.scala

```diff
@@ -101,6 +105,7 @@ class PendingTransactionsManager(
 
     case AddTransactions(signedTransactions) =>
       pendingTransactions.cleanUp()
+      log.debug("Added transactions: {}", signedTransactions.map(_.tx.hash.toHex))
```

Comment:
```suggestion
      log.debug("Added transactions: {}", signedTransactions.map(_.tx.hashAsHexString))
```

Reply:
why is this form better?

Reply:
only because it's already a part of SignedTransaction interface, otherwise it's a code duplication